e2e/features/Udacity.feature

```diff
@@ -15,15 +16,20 @@ Scenario: 1. The user should be able to use the search bar
 
    When the search bar is cleared
    Then the course count should equal to the remembered course count
-   
+
 Scenario: 2. The card details and the filter fields should work correctly
   Given the Udacity course page is opened
     
-   When the text "javascript" is typed into the search bar
+   When the course count is remembered
+    And the text "javascript" is typed into the search bar
    Then the selected filters field should contain the following filters:
         |javascript|
     And the course count should equal to the result counter
 
+   When the X "javascript" filter closer button is clicked
```

Comment:
The sentence would sound better this way: the X button is clicked for the "javascript" filter.